### DISCUSSION TITLE
社会与经济：决定开源软件生死的关键要素 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#35] [Announcements]
# 社会与经济：决定开源软件生死的关键要素 - 开源之道

https://opensourceway.community/posts/open-source-economic/moneitization-open-source-software/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

其实吧, 开源的收益, 仅仅从金钱角度分析, 很难说明普通人/VC/政府的...

FLOSS/自由开源软件 为什么屡禁不绝?

其实, 源自一个简单的原则: `千金难买我高兴`

参考: [phoebe的广播](https://www.douban.com/people/1556251/status/3869137582/?_i=526182190118dcb,2618285BXm1N7K&dt_dapp=1)
一个名为Mappiness project的项目研究哪些因素令人快乐/不快乐，主要方法是通过一个手机APP收集人们：在哪里？和谁在一起？在做什么？此刻快乐程度如何？

主要结论：

- 1. 最令人快乐的活动前三是：sex，运动，搞园艺；
- 2. 最令人不快乐的事情前二是：卧病在床、工作；
- 3. 跟爱人或朋友在一起提高快乐度，跟同事、孩子、熟人在一起不提高快乐度；
- 4.天气对快乐度影响没那么大，但25度以上的晴天确实令人快乐；
- 5. 在自然中令人快乐，尤其是在水边，尤尤其是在风景美丽的水边；
- 6. 使用社交媒体不会令人快乐。

所以, 人类发展出社会/工作/学习/... 等等自然界中根本就不存在的状态, 
都是反天性的;

而开源软件, 最大的收益, 其实并不是利益, 而是乐趣,
终于可以躲开那些被迫写出来的 `屎山` ,
在自己认同的项目中, 快乐的创作, 令自己感觉重新活过来, 享受创造的乐趣,
这是最强动力了...

这也吻合俺一直嗯哼的, 什么是共产主义社会?

```
劳动是第一需求
学习是第一娱乐
```

这种能快乐的劳动的场景, 是千金难买的;

所以, 开源软件生死关键问题, 并不在自身是否存在, 
而在社会大环境是否令普通程序员想维系一个有尊严的普通生活成本是否高到, 
必定榨掉所有精力, 每天基本不可能有时间/空间 进入欢乐世界....
